### ISSUE TITLE
Energy cards dont work with Mekanism Universal Cables

### ISSUE THREAD

**Issue by grav0r** [Direwolf20-MC/LaserIO#307]
### LaserIO version

1.9.11

### Minecraft Version

1.21

### (Neo)Forge Version

21.1.61

### Modpack & Version

Direwolf20 1.21 1.2.1

### Do you have optifine installed?

No

### Describe the issue

Energy cards dont work with Mekanism Universal Cables

### Steps to reproduce

Just try to extract or insert energy into or from a mekanism universal cable. In earlier versions it worked.

### Expected behaviour

Energy cards should transfer energy.

**Comment by Manbeartooth** [comment]
Any updates/progess on this issue?  
Its still there and the pack has been updated to the latest as of today, Nov 9th, so its running 1.4.1 now.

**Comment by llllDaturallll** [comment]
Bug still there
Tried with Pipez and Xnet to see if it's restricted to Mekanism but no, same issue with these mods

**Comment by volrathstormborn** [comment]
Yeah, it's all power pipe connections and I confirmed that it is not working with Flux Networks in 1.21.1 either.

**Comment by csuarezmtz** [comment]
Adding that this seems to also be the case with Flux Networks, just tried to export directly from a flux point but it wouldn't connect
